bundle/manifests/opentelemetry.io_opentelemetrycollectors.yaml

```diff
@@ -223,6 +223,11 @@ spec:
                   If MaxReplicas is set autoscaling is enabled.
                 format: int32
                 type: integer
+              minReplicas:
+                description: MinReplicas sets a lower bound to the autoscaling feature.  It
+                  must be at least 1
```

Comment:
Please remove this line break.

Reply:
@yuriolisa Is that possible?  This is generated code based on this comment, which is all on one line.  https://github.com/open-telemetry/opentelemetry-operator/blob/98739669d581e64458cd887609198489022f756e/apis/v1alpha1/opentelemetrycollector_types.go#L40

Reply:
It's possible that I overlooked this code.